.vscode/extensions.json

```diff
@@ -0,0 +1,17 @@
+{
+  "recommendations": [
+    "matklad.rust-analyzer",
```

Comment:
Resource monitor was useful in my case, as I like to know when my system is about to meltdown since rust builds are intensive :D
But in general, I agree; remove any that you deem unnecessary; I just copy-pasted what I use in most of my own projects, so a whole lot of them may be unnecessary.